src/components/docs/Avatar.md

```diff
@@ -8,48 +8,10 @@ const name = 'uthred';
 <Avatar image={image} name={name} size={35} />;
 ```
 
-A rounded Avatar
```

Comment:
do we no longer need this docs example?

Reply:
well there's no such thing as this as the avatar is already round...

Reply:
haha dope